### PR TITLE
alphabetically reorder acknowledgements list and make it more thorough

### PR DESCRIPTION
**Pull request by Sakurann**
For the awareness purposes, adding for review those who I added in the list. will merge when few approvals come in.

cc some of those whom I could not add as reviewers @c2bo, @TakahikoKawasaki, @cobward

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]
thanks 😄

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1630,7 +1630,7 @@ This specification registers the well-known URI defined in (#credential-issuer-w
 
 # Acknowledgements {#Acknowledgements}
 
-We would like to thank Daniel Fett, Brian Campbell, Joseph Heenan, Daniel McGrogan, David Chadwick, John Bradley, Mark Haine, Alen Horvat, Michael B. Jones, Kenichi Nakamura, Oliver Terbu and David Waite for their valuable contributions to this specification.
+We would like to thank Alen Horvat, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Gabe Cohen, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable contributions to this specification.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1633]:
```suggestion
We would like to thank Alen Horvat, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Gabe Cohen, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable contributions to this specification.
```

**Review by bifurcation — APPROVED** [review]

**Review by TimoGlastra — APPROVED** [review]

**Review by c2bo — APPROVED** [review]

**Review by paulbastian — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by peppelinux — APPROVED** [review]
Thanks

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -1630,7 +1630,7 @@ This specification registers the well-known URI defined in (#credential-issuer-w
 
 # Acknowledgements {#Acknowledgements}
 
-We would like to thank Daniel Fett, Brian Campbell, Joseph Heenan, Daniel McGrogan, David Chadwick, John Bradley, Mark Haine, Alen Horvat, Michael B. Jones, Kenichi Nakamura, Oliver Terbu and David Waite for their valuable contributions to this specification.
+We would like to thank Alen Horvat, Andrii Deinega, Arjen van Veen, Brian Campbell, Christian Bormann, David Chadwick, Daniel Fett, Daniel McGrogan, David Waite, Gabe Cohen, George Fletcher, Giuseppe De Marco, Jacob Ideskog, Jacob Ward, Jeremie Miller, John Bradley, Joseph Heenan, Judith Kahrer, Kenichi Nakamura, Mark Dobrinic, Mark Haine, Markus Kreusch, Michael B. Jones, Nat Sakimura, Niels Klomp, Oliver Terbu, Paul Bastian, Pedro Felix, Richard Barnes, Roland Hedberg, Rolson Quadras, Takahiko Kawasaki, Timo Glasta, Tom Jones, Vittorio Bertocci for their valuable contributions to this specification.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1633]:
```suggestion
We would like to thank Paul Bastian, Vittorio Bertocci, Christian Bormann, John Bradley, Brian Campbell, Gabe Cohen, David Chadwick, Andrii Deinega, Giuseppe De Marco, Mark Dobrinic, Daniel Fett, Pedro Felix, George Fletcher, Timo Glasta, Mark Haine, Fabian Hauck, Roland Hedberg, Joseph Heenan, Alen Horvat, Andrew Hughes, Jacob Ideskog, Edmund Jay, Michael B. Jones, Tom Jones, Judith Kahrer, Takahiko Kawasaki, Niels Klomp, Ronald Koenig, Markus Kreusch, Adam Lemmon, Daniel McGrogan, Jeremie Miller, Kenichi Nakamura, Rolson Quadras, Nat Sakimura, Oliver Terbu, Arjen van Veen, David Waite, Jacob Ward for their valuable feedback and contributions to this specification.
```

**Review by jogu — APPROVED** [review]

**Pull request merged**